### ISSUE TITLE
Reducing palette stepwise

### ISSUE THREAD

**Issue by lindehell** [maxcleme/beadifier#28]
Would it be a good feature to have an option to reduce colors by a given number instead of percent?
I usually work with a huge palette of colors and more often than not there is a lot of background that take up 50-70% of the total bead count. The result is I end up with just a handful of colors  from the reduce under 10 % option and me loosing all the fine details in the picture. 

Consider adding a button where you reduce the 10 least used colors or perhaps even a slider so you can reduce by any number you want.

**Comment by lindehell** [comment]
Another option would be to tag certain colors as background and reduce 10 from the remaining.
Can't decide in my head which I would prefer. 

Just a thought :)

**Comment by maxcleme** [comment]
@lindehell I understand your point and it makes absolute sense, do you have any insight on how it could be integrated to the current UI ? 

Adding a slider would not overcharge the UI, however having the possibility to "lock" certain colors could definitely overcharge the UI.

Anyway, I need to think about it, but I think it's a good idea :)

**Comment by lindehell** [comment]
I am just the pixelartist with ideas. I would be crazy satisfied with the slider option. Understand the other one might be complicated. Thanks for listening to my ideas :)

**Comment by lindehell** [comment]
Coming back to this oldie. I have attached a screenshot of what many of my project look like when starting out.

Would it really overload the UI if you had a little eye icon or a checkbox by the colors so you could say "Yes, I know you are there. I love you, but let's focus on the rest". The usage screen is very hard to work with right now.

Another option would be a slider/checkbox on the side with "Remove/Hide" to determine the action when clicking a column/pie piece.

What do you think?
![Screenshot_20201212-125648-01](https://user-images.githubusercontent.com/13117403/101983309-8fbae900-3c7a-11eb-9b2e-efb06fc42f3b.jpeg)

**Comment by maxcleme** [comment]
@lindehell

100% agree that this usage tab needs to be reworked from the group up.

Moreover, using mobile device I can feel your pain. This was design at the very beginning of the project and should be rethink entirely. 

IMO we want to pin point two importants information here : distribution and amount of beads. 

May I ask you to think about some mock up? Any idea what is your ideal screen?

**Comment by lindehell** [comment]
I personally don't use the top "pie" diagram. If I could swipe and zoom in the lower one (so I could read the on the X-axis) I would be more than happy. Regarding the hide option would it be possible to have a drop-down menu or a slider of some sort to switch between Removing and hiding from the diagram. The hidden ones could show up in a list so one could get them back.

Something like this (came up with the list while writing so it's not in there):
![PicsArt_01-25-06 58 10](https://user-images.githubusercontent.com/13117403/105746819-3532cf00-5f40-11eb-82a2-781ee1c3fd90.jpg)

Sorry for not being more active, but you are doing wonderful work here.

**Comment by maxcleme** [comment]
@lindehell 

Sorry for late response.

As already state at the beginning of this issue, this "usage" tab needs to be reworked from scratch. 

I think colors reduction button should move somewhere else (would make more sense in the "palette" tab TBH).
This ugly "Undo" button should be later handle with a more global "Undo / Redo" feature.

Then for the usage tab, I would opt for displaying only 1 chart/plot at a time, having a dropdown to choose which one you want. They all display the same things, distribution, however depending on your project you might prefer the "radar" or the "barchart".

Since it was development for desktop browser first, I think it might also be the right time to change the library or update it to have better mobile support (and better responsive handling as a whole). Usually, navigating through chart/plot using smartphone is such a pain, I hope we could find something more suitable out there.

I'm running out of time these past months, sorry for but being more active right here.

Cheers.

**Comment by lindehell** [comment]
Total makeover of the usage tab sounds great. No worries about time, guessing this is not your cash cow. I'm in it for the long haul.

**Comment by maxcleme** [comment]
> guessing this is not your cash cow

True, I do this for free, and it will remains free and without ads as long as I'm the main maintainer.
